README.md

```diff
@@ -94,7 +94,11 @@ This file simulates the Cloud Foundry environment variables required for your ap
 
 ## Requirements and Setup
 
-*Insert a short description what is required to get your project running...*
+To get your project running, ensure you have the following prerequisites:
+
+- Java 17 or higher installed
+- A user-provided service instance for Audit Log NG created in your Cloud Foundry space
```

Comment:
_Audit Log NG_ -> is there already a brand approved name for this service? Is it a different service than _SAP Audit Log service_, or just the usage a newer version of its API?

What SAP help page better describes the service? Is it https://help.sap.com/docs/sap-audit-log?
We should link to the best public documentation we have here.